File: python/p15.py
x=int(input("Enter x:"))
y=int(input("Enter y:"))
if(x>=0 and y>=0):
    if(x>y):
        print("1st")
    else:
        print("2nd")
elif(x<0 and y>=0):
    if(-x>y):
        print("4th")
    else:
        print("3rd")
elif(x<0 and y<0):
    if(x>y):
        print("6th")
    else:
        print("5th")
else:
    if(x>-y):
        print("8th")
    else:
        print("7th")

File: Manual converted to C++/p15.cpp

#include<iostream>
using namespace std;
int main(){
    int x,y;
    cout<<"Enter x:";
    cin>>x; 
    cout<<"Enter y:";
    cin>>y; 
    if(x>=0 && y>=0){
        if(x>y)
            cout<<"1st";
        else
            cout<<"2nd";
    }
    else if(x<0 && y>=0){
        if(-x>y)
            cout<<"4th";
        else
            cout<<"3rd";
    }
    else if(x<0 && y<0){
        if(x>y)
            cout<<"6th";
        else
            cout<<"5th";
    }
    else{
        if(x>-y)
            cout<<"8th";
        else
            cout<<"7th";
    }
    return 0;
}
